src/js/plugin/sdk/pipeline/PipelineModel.js

```diff
@@ -0,0 +1,46 @@
+
+export default class PipelineModel {
+
+  constructor() {
+    this.pipelines = new Map();
+  };
+
+  applyPipeline(pipelineName, filterParameters) {
+    if (!this.pipelines.has(pipelineName)) {
+      return filterParameters;
+    }
+    return this.pipelines.get(pipelineName)
+      .reduce((accumulator, pipelineCallback) => {
```

Comment:
🎨  I'd rename `pipelineCallback` to `operator` to stick with the general naming scheme.

Reply:
Done.